chess/0-chess-moves/getting-started.md

```diff
@@ -8,7 +8,7 @@ Take the following steps to set up your Chess project.
 
 ## Adding the Template Chess Project
 
-1. Unzip the [starter-code/chess.zip](starter-code/chess.zip) file to the directory where you cloned the Chess GitHub Repository.
+1. Unzip the [starter-code/chess.zip](starter-code/chess.zip) file to the directory where you cloned the Chess GitHub Repository. You should run this command from inside of your your personal projects directory. 
```

Comment:
@Truthwatcher64  - There is a typo `your your`. I actually find the term `personal projects directory` confusing. The previous sentence conveys the proper location. The addition of the clarification makes me think I should be running it in the root of all my course work, or maybe where I simply do all my coding.

Reply:
I agree Lee. I thought that sentence might create confusion too (although I didn't notice the extra 'your'). I should have asked for a change here before approving the PR.